apps/webservice/src/app/[workspaceSlug]/_components/target-drawer/relationships/RelationshipsDiagram.tsx

```diff
@@ -100,7 +180,7 @@ const DepEdge: React.FC<EdgeProps> = ({
     <>
       <BaseEdge
         path={edgePath}
-        markerEnd={markerEnd}
+        markerStart={markerEnd}
```

Comment:
```suggestion
        markerStart={arrow}
```

weird to call it markerend, confusing

Reply:
`{ type: MarkerType.Arrow, color: colors.neutral[700] },`

Reply:
these names come from edge props whcih are intrinsict to the reactflow library